src/static/stylesheets/defaults.scss

```diff
@@ -30,20 +30,101 @@
   cursor: pointer;
 }
 
-.actions-wrapper {
-  .actions-container {
-    &:not(:last-child) {
-      margin-bottom: $elv-spacing-s;
-    }
-  }
-}
-
 .actions-container {
+  display: flex;
   min-height: 1.7rem;
   position: relative;
 
-  &.manage-actions {
-    margin-bottom: $elv-spacing-m;
+  .left-action-buttons,
+  .right-action-buttons {
+    display: inline-flex;
+    height: 100%;
+  }
+
+  .right-action-buttons {
+    position: absolute;
+    right: 0;
+  }
+
+  .more-options-container {
+    display: inline-block;
+
+    .more-options-text {
+      padding-right: 15px;
+      position: relative;
+
+      &::after {
+        content: "\25BC";
+        position: absolute;
+        right: 0;
+        transform: scale(1, 0.5);
+      }
+    }
+  }
+
+  .more-options-menu {
+    background-color: $elv-color-bg-white;
+    border-radius: 4px;
```

Comment:
Since the box borders the button, I think the border radius should be 0 so there's no gap between them